pkg/dispatchcli/cmd/update.go

```diff
@@ -0,0 +1,46 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package cmd
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+	"github.com/vmware/dispatch/pkg/dispatchcli/i18n"
+)
+
+var (
+	updateLong = i18n.T(`Create a resource. See subcommands for resources that can be created.`)
+
+	// TODO: Add examples
+	updateExample = i18n.T(``)
+	//file          = i18n.T(``)
+)
+
+func NewCmdUpdate(out io.Writer, errOut io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "update",
+		Short:   i18n.T("Update resources."),
+		Long:    updateLong,
+		Example: updateExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			if file == "" {
+				runHelp(cmd, args)
+				return
+			}
+
+			updateMap := map[string]modelAction{
+				"secret": CallUpdateSecret,
+			}
+
+			err := importFile(out, errOut, cmd, args, updateMap)
```

Comment:
Does this actually work?  It doesn't seem as though you've added a file argument to the update command.

Reply:
I admit that I took this from the code that sets up the create command. Now that I look at this, I'm just as confused as you are. It compiles and runs properly, but I can't tell where that `file` variable comes from.

Reply:
Actually I have an idea... I believe file is declared as a package scoped variable in the create.go file. In that file it is declared as `file = i18n.T(\`\`)`